remark/mermaid/mermaid.config.json

```diff
@@ -7,6 +7,16 @@
     "diagramPadding": 1,
     "useMaxWidth": false
   },
+  "gantt": {
```

Comment:
Just confirming, I'm assuming we've come up with these values based on how it looks visually?

Reply:
Yeah, I entirely made them up because Mermaid doesn't structure the Gantt diagram in a way that is conducive to how Braid/we like to represent spacing. If we try to get rid of this extra padding, Mermaid's renderer will squish/or cut off the actual diagram, so we'd have to raise the change upstream.